qa/workunits/rbd/krbd_watch_errors_exclusive.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -ex
+set -o pipefail
+
+readonly IMAGE_NAME="watch-errors-exclusive-test"
+
+rbd create -s 1G --image-feature exclusive-lock,object-map "${IMAGE_NAME}"
+
+# induce a watch error every 30 seconds
+dev="$(sudo rbd device map -o exclusive,osdkeepalive=60 "${IMAGE_NAME}")"
+dev_id="${dev#/dev/rbd}"
+
+sudo dmesg -C
+
```

Comment:
So we don't have to constantly refresh as done here https://github.com/ceph/ceph/commit/237aa221ebad429457a621d2e38cfdf0025e38f9 ?

Reply:
No, this isn't required for reproducing this one.